core/src/core-plugin-definitions.ts

```diff
@@ -393,7 +399,7 @@ export interface CameraPhoto {
    */
   exif?: any;
   /**
-   * The format of the image. Currently, only "jpeg" is supported.
+   * The format of the image, ex: jpeg, png, gif.
```

Comment:
iOS and Android only allow `jpeg`, should be noted that `png` and `gif` are web only, and I think `gif` will only work if using the file input, not pwa elements.